lib/metrics/StatsModel.js

```diff
@@ -17,7 +17,10 @@ class StatsModel extends StatsClient {
     * @return {array} converted array
     */
     _zip(arrays) {
-        return arrays[0].map((_, i) => arrays.map(a => a[i]));
+        if (arrays.length > 0 && arrays.every(a => Array.isArray(a))) {
+            return arrays[0].map((_, i) => arrays.map(a => a[i]));
+        }
+        return arrays;
```

Comment:
Can you comment why you return `arrays` here if items are not arrays? I think this is to support the zero-fill case though not crystal clear for me.

Reply:
I actually think I should return an empty array (or validate after `_zip` returns). The `StatsClient.getStats` on error will return an integer. The original comment says that Redis for stats isn't a critical component, so the default response is a 0.
Or if no redis keys, `_getCount` will return an array of 0's.